cpp/src/arrow/io/hdfs_internal.cc

```diff
@@ -212,10 +219,16 @@ Result<std::vector<PlatformFilename>> get_potential_libjvm_paths() {
           "/usr/local/lib/jvm/java-6-oracle",         // alt ubuntu
           "/usr/lib/jvm/default",                     // alt centos
           "/usr/java/latest",                         // alt centos
+          "/usr/local/lib/jvm/java-1.8.0-openjdk-"+prefix_arch,  // debian distros / arm64 architecture
+          "/usr/lib/jvm/java-1.8.0-openjdk-"+prefix_arch,        // debian distros / arm64 architecture
+          "/usr/local/lib/jvm/java-1.7.0-openjdk-"+prefix_arch,  // debian distros / arm64 architecture
+          "/usr/lib/jvm/java-1.7.0-openjdk-"+prefix_arch,        // debian distros / arm64 architecture
+          "/usr/local/lib/jvm/java-1.6.0-openjdk-"+prefix_arch,  // debian distros / arm64 architecture
+          "/usr/lib/jvm/java-1.6.0-openjdk-"+prefix_arch        // debian distros / arm64 architecture
```

Comment:
I don't think these added lines are necessary.

Reply:
Maybe for those who use openjdk?  Anyway, I had removed these line, thtank you for your reminding.